### ISSUE TITLE
Add missing "T1" and "N." 2525C modifiers

### ISSUE THREAD

**Issue by vandytx** [missioncommand/mil-sym-js#36]
Could you please add the following modifiers in the next release?: 

{
"ID": "G*G*GLF---****X",
"G": "line",
"DC": "1",
"MXP": "10000",
"MNP": "2",
*``` "M": "N.", ```*
"D": "Forward Line of Own Troops",
"H": "2.X.2.1.2.2",
},
{
"ID": "G*F*LCF---****X",
"G": "line",
"DC": "1",
"MXP": "10000",
"MNP": "2",
*```"M": "T.T1.W.W1.",```*
"D": "Fire Support Coordination Line",
"H": "2.X.4.2.2.1",
},
{
"ID": "G*F*LCC---****X",
"G": "line",
"DC": "1",
"MXP": "10000",
"MNP": "2",
*```"M": "T.T1.W.W1.",```*
"D": "Coordinated Fire Line (CFL)",
"H": "2.X.4.2.2.2",
},
{
"ID": "G*F*LCR---****X",
"G": "line",
"DC": "1",
"MXP": "10000",
"MNP": "2",
*```"M": "T.T1.W.W1.",```*
"D": "Restrictive Fire Line (RFL)",
"H": "2.X.4.2.2.4",
},

Thanks.

**Comment by michael-spinelli** [comment]
T1 is not part of those symbols.  But I'll take a look at FLOT.

![image](https://user-images.githubusercontent.com/5972575/57776520-b6efb400-76ed-11e9-9315-ff8535daf8c8.png)

**Comment by vandytx** [comment]
Thanks Michael. I had not looked at the USAS document. I'm guessing there's no 2525C version of this document, meaning this advice applies to 2525C as well?

**Comment by michael-spinelli** [comment]
Correct, there isn't currently a 2525C based version of the USAS so assume this applies to 2525C as well.
It's a clarification because the template and example keep getting people confused about this symbol.

I've made the update for FLOT so it'll be in there next time I do a release.

**Comment by vandytx** [comment]
Perfect. Thank you!
